cmd/clusterctl/cmd/upgrade_apply.go

```diff
@@ -55,10 +55,9 @@ var upgradeApplyCmd = &cobra.Command{
 		in order to guarantee the proper functioning of the management cluster.
 
  		Specifying the provider using namespace/name:version is deprecated and will be dropped in a future release.`),
-
 	Example: templates.Examples(`
 		# Upgrades all the providers in the management cluster to the latest version available which is compliant
-		# to the v1alpha4 API Version of Cluster API (contract).
+		# to the v1alpha4 Cluster API contract version.
 		clusterctl upgrade apply --contract v1alpha4
```

Comment:
We don't always have to keep this up-to-date. But while we're here, maybe change the example to v1beta2?

(similar below in the flag description)